store/user.js

```diff
@@ -6,6 +6,20 @@ export const state = () => ({
   list: {}
 })
 
+function getUserByID(state, id) {
+  let user = null
+
+  Object.keys(state.list).forEach(key => {
+    const item = state.list[key]
+
+    if (parseInt(key) === parseInt(id)) {
+      user = item
+    }
+  })
+
+  return user
+}
+
```

Comment:
This loop could be simplified as `state.list` is an object keyed by the id, so should be the same as:
```js
function getUserByID(state, id) {
  return state.list[id]
}
```
Try it first though...

Reply:
I decided not to touch this from what it was before.  But happy to update if everyone agrees.  It certainly looks a bit (much) nicer.

Reply:
We can let @edwh decide :smile:

Reply:
Well it seems to work ok but I'll give it a proper test on Monday.  It is much more readable!